### DISCUSSION TITLE
Centraliser les outils que l'on peut utiliser

### DISCUSSION THREAD

**Discussion by sam-pires** [betagouv/cap#11] [Besoin]
### Besoin

```bash
Bonjour,

C'est compliqué de savoir quels outils on a le droit ou non d'utiliser, et ceux auxquels on a accès via nos incubateurs ou Beta.

Ce serait utile d'avoir une liste claire et les liens vers les procédures pour y accéder s'ils sont déjà gérés par Beta
```

### Solutions envisageables

```bash
1. Page web/annuaire qui liste les outils par catégorie, y compris ceux qui ne sont pas autorisés
```

### Informations complémentaires

- [ ] J'avais déjà entendu ce besoin au sein de la communauté beta.gouv
- [X] Je me porte volontaire pour participer à la mise en place de la future solution choisie

**Comment by revolunet** [comment]
Je propose qu'on ajoute une liste des "produits recommandés" et les bonnes pratiques associées dans la [doc sillon](https://sillon.incubateur.net/)

**Reply by revolunet** [reply]
au final la liste des outils proposés chez beta est dans la doc beta : https://doc.incubateur.net/communaute/travailler-a-beta-gouv/jutilise-les-outils-de-la-communaute

**Reply by adipasquale** [reply]
https://grist.incubateur.net/ n’est pas listé dans cette doc beta alors que c’est une super alternative à google sheets 👍

**Reply by revolunet** [reply]
fixed: https://doc.incubateur.net/communaute/travailler-a-beta-gouv/jutilise-les-outils-de-la-communaute/grist

**Comment by revolunet** [comment]
je close car la doc est censée être à jour sur ce sujet. réouvrir si besoin
